yearzero/models/Weapon.js

```diff
@@ -53,9 +54,9 @@ class Weapon extends BaseGear {
 		});
 	}
 
-	toString() {
-		let str = `**${this.name}:** Gear Bonus **+${this.bonus}**, weapon damage **${this.damage}**`
-		+ `, \`${this.range.toUpperCase()}\` range${this.comment ? `, *${this.comment}*` : ''}`;
+	toString(lang = 'en') {
```

Comment:
Is it possible to add parameters to this function? It's overriding the default object's `toString()`.

Reply:
Yes, this works because with the default value it becomes an optional parameter. So if the default toString() is called it uses the value 'en'.